app/src/main/java/org/fossasia/openevent/general/search/SearchResultsFragment.kt

```diff
@@ -46,6 +46,8 @@ import org.koin.androidx.viewmodel.ext.android.viewModel
 import timber.log.Timber
 import org.fossasia.openevent.general.utils.Utils.setToolbar
 
+const val INITIAL_FETCHING_EVENTS: Int = 1
```

Comment:
Unneeded

Reply:
I'm using this because onCreate is being called on rotation. I know it shouldn't be but it is.
I really don't know why.

Reply:
then how can we check that load events func is being called from onCreate everytime on rotation?

Reply:
Just check if the events are not empty

Reply:
And when the user clicks on the chip, it will also return from there.

Reply:
Use a parameter in the function in case of chip click

Reply:
done..